packages/playground/src/Layout/Layout.tsx

```diff
@@ -36,12 +38,125 @@ NavButtonComponent.displayName = "NavButtonComponent";
 const NavButton = createLink(NavButtonComponent);
 
 export function Layout({ children }: { children?: ReactNode }) {
-  const { isWorkflowsEnabled, isTracingEnabled, shouldShowTopNav } =
-    useStudioStore(
-      "isWorkflowsEnabled",
-      "isTracingEnabled",
-      "shouldShowTopNav",
-    );
+  const {
+    isWorkflowsEnabled,
+    isTracingEnabled,
+    shouldShowTopNav,
+    setShortcutsOpen,
+    setActiveRequestsPanelTab,
+    visibleRequestsPanelTabs,
+    setCommandBarOpen,
+  } = useStudioStore(
+    "isWorkflowsEnabled",
+    "isTracingEnabled",
+    "shouldShowTopNav",
+    "setShortcutsOpen",
+    "setActiveRequestsPanelTab",
+    "visibleRequestsPanelTabs",
+    "setCommandBarOpen",
+  );
+
+  const formRef = useRef<HTMLFormElement>(null);
+
+  useHotkeys(
```

Comment:
okay so a lot of these hotkeys are specific to the playground page - like 

- requestSubmit
- go to docs
- go to request params
- etc

might be good to move those hooks back to the playground index page